exp/api/v1beta1/azuremanagedmachinepool_webhook.go

```diff
@@ -271,7 +223,7 @@ func (m *AzureManagedMachinePool) validateLastSystemNodePool(cli client.Client)
 		return err
 	}
 
-	if len(ammpList.Items) <= 1 {
+	if len(ammpList.Items) < 1 {
```

Comment:
seems like we're changing core validation logic, is this fixing a bug? If so, can you please split the commits?

Reply:
yeah, this was an oversight, thanks!